.ci-builds/.azure-valgrind.yml

```diff
@@ -61,6 +61,10 @@ jobs:
         cd icu4c/source/test/cintltst && LD_LIBRARY_PATH=../../lib:../../stubdata:../../tools/ctestfw:$LD_LIBRARY_PATH valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --show-reachable=yes ./cintltst
       displayName: 'Valgrind cintltst'
       timeoutInMinutes: 15
+    - script: |
```

Comment:
The valgrind CI pipeline doesn't run on normal PRs and is really only for testing with Valgrind.
I'd suggest adding this to the "regular" CI pipeline instead.